docs/source/optimisation.rst

```diff
@@ -59,6 +59,9 @@ print to screen of the status of the optimisation.
    :members:
 .. autoclass:: cil.optimisation.algorithms.SIRT
    :members: update, update_objective
+.. autoclass:: cil.optimisation.algorithms.ISTA
+   :members:
+   :special-members:   
```

Comment:
I don't think we want `special-members` as then you get the `init` doc string. But check how the docs look.